src/main/java/com/lpvs/service/LPVSQueueService.java

```diff
@@ -146,14 +146,9 @@ public BlockingDeque<LPVSQueue> getQueue() {
     public void checkForQueue() throws InterruptedException {
         log.debug("Checking for previous queue");
         List<LPVSQueue> webhookConfigList = queueRepository.getQueueList();
-        if (webhookConfigList.size() > 0) {
-            LPVSQueue webhookConfig = getLatestScan(webhookConfigList);
-            webhookConfig.setAttempts(webhookConfig.getAttempts() + 1);
-            queueRepository.save(webhookConfig);
-        }
         for (LPVSQueue webhook : webhookConfigList) {
             log.info("PROCESSING WebHook id = " + webhook.getId());
-            if (webhook.getAttempts() > maxAttempts) {
+            if (webhook.getAttempts() < maxAttempts) {
                 LPVSPullRequest pullRequest = new LPVSPullRequest();
                 pullRequest.setPullRequestUrl(webhook.getPullRequestUrl());
                 pullRequest.setUser(webhook.getUserId());
```

Comment:
The same webhook processing should be added to exception processing before delete().